### PR TITLE
Features 1

### PR DESCRIPTION
**Pull request by DJ-MrJay**
My Portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                </div>
+            </header>
+
+            <div class="social-icons">
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-facebook"></i>
+                <i class="fab fa-linkedin-in"></i>
+                <i class="fab fa-github"></i>
+                <i class="fab fa-youtube"></i>
+            </div>
```
Comment (Ugboaja-Uchechi) [inline index.html:49]: - [ ] Kindly change your `div` tag to a `ul` tag and wrap each of your `icons` in an `li` tag so your code can be semantic which would also make your site better for screen readers. You can read more about semantic HTML in this [article](https://www.w3schools.com/html/html5_semantic_elements.asp) 🙏

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    background: #ffffff;
+    height: 50px;
+    max-width: 375px;
+}
+
+.wrapper {
+    max-width: 327px;
+    margin: 0 auto;
+}
+
```
Comment (Ugboaja-Uchechi) [inline styles.css:23]: - [ ] Please make sure your design looks as close as possible to the figma design.

Figma Design | Your Design
------------- | -------------
Figma design  |  User design

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @DJ-MrJay , 

## Changes Required

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Your repo looks professional 👏 
- Great job setting up your linters 👏 

## Required Changes ♻️

- Please make sure all linters are passing. To fix stylelint errors run `npx stylelint "**/*.{css,scss}"` and then fix the errors manually. To fix them automatically run `npx stylelint "**/*.{css,scss}" --fix`.
- Please add a project description to your pull request so that your project looks more professional 🙏 .
- Please make sure your README  is updated and has all the necessary sections needed so that any one who goes to your repo would know what you worked on. You can follow the [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) and adjust it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @DJ-MrJay. 

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages  ✔️ 
- Professional Pull request ✔️
- Professional Readme ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORODN` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vic778 — APPROVED** [review]
Hi @DJ-MrJay   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Pull request merged**